src/app/_pages/housing-available-add/housing-available-add.page.scss

```diff
@@ -0,0 +1,26 @@
+.range {
+  margin-bottom: 1.176rem;
+  .range-items {
+    display: flex;
+    flex-direction: row;
+  }
+  .range-divider {
+    font-size: 2rem;
+    margin-left: 1rem;
+    margin-right: 1rem;
+    padding-top: 1rem;
+  }
+  > .mat-form-field-subscript-wrapper {
+    margin-top: -1.176rem;
+    // margin-bottom: 1.176rem;
+  }
+}
```

Comment:
don't think we use range on this page, so we could remove this to keep the styles file clean.